oteapi/datacache/datacache.py

```diff
@@ -225,34 +178,35 @@ def getfile(  # pylint: disable=too-many-arguments
         """Write the value for `key` to file and return the filename.
 
         The file is created in the default directory for temporary
-        files (which can be controlled by the `TEMPDIR`, `TEMP` or `TMP`
+        files (which can be controlled by the TEMPDIR, TEMP or TMP
         environment variables). It is readable and writable only for
         the current user.
 
+        This method is intended to be used in a `with` statement, to
+        automatically delete the file when leaving the context.
+
         Example:
-            This method is intended to be used in a `with` statement, to
-            automatically delete the file when leaving the context:
 
-            ```python
-            cache = DataCache()
-            with cache.getfile('mykey') as filename:
-                # do something with filename...
-            # filename is deleted
-            ```
+        ```python
+        cache = DataCache()
+        with cache.getfile('mykey') as filename:
+            # do something with filename...
+        # filename is deleted
+        ```
```

Comment:
```suggestion
            ```python
            cache = DataCache()
            with cache.getfile('mykey') as filename:
                # do something with filename...
            # filename is deleted
            ```
```